src/ucs/sys/sock.c

```diff
@@ -342,3 +342,104 @@ int ucs_sockaddr_is_equal(const struct sockaddr *sa1,
     }
     return result;
 }
+
+static void ucs_sockaddr_khash_jenkin_mix(uint32_t *hj_i,
+                                          uint32_t *hj_j,
+                                          uint32_t *hj_val)
+{
+    uint32_t a = *hj_i;
+    uint32_t b = *hj_j;
+    uint32_t c = *hj_val;
+    
+    a -= b; a -= c; a ^= (c >> 13);
```

Comment:
can we reuse ucs_crc16()?

Reply:
yes, it makes sense to me. I don't know about this. Closing this PR